app/helpers/report_helper/timer.rb

```diff
@@ -0,0 +1,70 @@
+ReportHelper::Timer = Struct.new(
+  :typ,
+  :months,
+  :weeks,
+  :days,
+  :hours,
+  :start_date,
+  :start_hour,
+  :start_min,
+) do
+  def initialize(typ = nil, months = 1, weeks = 1, days = 1, hours = 1, start_date = nil, start_hour = '00', start_min = '00')
+    super
+  end
```

Comment:
At this rate, you might want to just make a regular class with attr_accessor instead of a Struct.  @matthewd What is the general consensus of the Ruby community on Struct vs Class?